### ISSUE TITLE
[Suggestion] Add a forge table like terraria to reforge

### ISSUE THREAD

**Issue by ghost** [Draylar/tiered#34]
maybe cost gold or emerald to reforge

**Comment by Shadowdragon019** [comment]
Yes! I quite want this! I'd suggest it would require the item it's repaired with to 'Re-Forge' it.

**Comment by githubbingitup** [comment]
Yeah, a way to reroll stats on items would be great.

**Comment by ElectricSeal20** [comment]
Is there any way to edit the nbt with in-game commands to kind of achieve this re-roll functionality?
